Makefile

```diff
@@ -0,0 +1,16 @@
+# Override the arch with `make ARCH=i386`
+ARCH ?= $(shell flatpak --default-arch)
+REPO ?= repo
+
+# Convenience to install deps. If you already have a gnome just install from that!
+install-deps:
+	flatpak --user remote-add --from gnome https://sdk.gnome.org/gnome.flatpakrepo
+	flatpak --user install gnome org.freedesktop.Platform//1.4 org.freedesktop.Sdk//1.4
+
+all: ${REPO}
```

Comment:
`all` should be the first rule; otherwise `make` with no target will pick `install-deps`